api/v3/ContactType.php

```diff
@@ -75,7 +75,6 @@ function civicrm_api3_contact_type_create($params) {
  *   Array of matching contact_types
  */
 function civicrm_api3_contact_type_get($params) {
-  civicrm_api3_verify_mandatory($params);
```

Comment:
Does mandatory params happen as part of basic get?

Reply:
it happens before it reaches that function in the api wrapper